### PR TITLE
Enable pending cops

### PR DESCRIPTION
**Pull request by thefloweringash**

### CODE REVIEW

File: rubocop.yml [hunk 1]
```diff
@@ -173,3 +173,29 @@ Layout/FirstHashElementIndentation:
...
+# Pending Cops as of 0.87.1
+Layout/EmptyLinesAroundAttributeAccessor:
+  Enabled: true
+Layout/SpaceAroundMethodCallOperator:
+  Enabled: true
+Lint/DeprecatedOpenSSLConstant:
+  Enabled: true
+Lint/MixedRegexpCaptureTypes:
+  Enabled: true
+Style/AccessorGrouping: # maybe
```
Comment (thefloweringash) [inline rubocop.yml]: This one may be contentious. Separating attributes can be useful to add comments. I see only a single instance of this in our main code base though.

File: rubocop.yml [hunk 2]
```diff
@@ -173,3 +173,29 @@ Layout/FirstHashElementIndentation:
...
+  Enabled: true
+Style/BisectedAttrAccessor:
+  Enabled: true
+Style/ExponentialNotation:
+  Enabled: true
+Style/RedundantAssignment:
```
Comment (thefloweringash) [inline rubocop.yml]: This one I expected to have many violations, but I only saw one and it didn't seem to have any justification.

File: rubocop.yml [hunk 3]
```diff
@@ -173,3 +173,29 @@ Layout/FirstHashElementIndentation:
   # This one in particular forbids what Emacs does by default for multi-line
   # hash entries
   Enabled: false
+
+# Pending Cops as of 0.87.1
```
Comment (thefloweringash) [inline rubocop.yml]: This file is getting long. Some day we'll need to organize it more carefully. This section is expected to be removed as of the next version bump when the pending cops are all enabled.

**Review by thefloweringash — COMMENTED** [review]
I checked all the cops seemed reasonable before enabling. If we're of the opinion that new cops are likely to be acceptable we can opt in by default with

```yaml
AllCops:
  NewCops: enable
```

**Comment by chrisandreae** [comment]
Given the advisory way we use Rubocop, I'm largely in favour of enabling new cops until/unless they prove bad for us. As such, I think I'd be happy with `NewCops: enable`. What's your opinion?

File: rubocop.yml (hunk 1 above)
Comment (chrisandreae) [reply rubocop.yml]: I take your point here, I think I'd prefer the rule to be "accessor declarations may not be separated by any expressions that are not declaring accessors". Sadly I don't think that's an option?

**Comment by thefloweringash** [comment]
> I think I'd be happy with `NewCops: enable`. What's your opinion?

Works for me.

**Pull request merged**